src/SeleniumLibrary/keywords/element.py

```diff
@@ -112,21 +113,42 @@ def page_should_contain_element(self, locator, message=None, loglevel='INFO'):
         The ``message`` argument can be used to override the default error
         message.
 
+        The ``limit`` argument can used to define how many elements the
+        page should contain. When ``limit`` is `None` page can contain one or
```

Comment:
Should have double backticks around `None`. Might be a good idea to add `(default)` after `None` to make it explicit that this is the default behavior.